Dockerfile.deployment

```diff
@@ -44,12 +43,8 @@ RUN composer install --no-autoloader --no-dev
 COPY package.json yarn.lock ./
 RUN yarnpkg
 
-RUN rm -f /var/log/nginx/access.log /var/log/nginx/error.log && \
```

Comment:
this and wrong `/var/lib/nginx` owner seem to cause this error? 🤔 

(the original lines were made so the logs are passed to docker, taken from [nginx official docker image](https://github.com/nginxinc/docker-nginx/blob/master/Dockerfile-debian.template#L88))
```
~/osu/osu-web-docker$ docker run nanaya/osu-web assets
nginx: [alert] could not open error log file: open() "/var/log/nginx/error.log" failed (13: Permission denied)
2022/04/27 03:58:06 [emerg] 1#1: mkdir() "/var/lib/nginx/body" failed (13: Permission denied)
```

Reply:
the commit in which i removed that was completely unrelated :thinking: